makefile

```diff
@@ -129,6 +132,12 @@ run: kill frontend
 		done
 	make superposition -e DOCKER_DNS=$(DOCKER_DNS)
```

Comment:
`make superposition$(FAST) -e DOCKER_DNS=$(DOCKER_DNS)`

Reply:
We can modify the existing run target to achieve this, making the Makefile more DRY .

Reply:
Sure, this is something I added for testing. We can remove it, fast mode isn't required to be on always